.gitignore

```
@@ -1,12 +1,139 @@
-*.py[co]
+# setuptools-scm
+_version.py
 
-build
-dist
-iris_sample_data.egg-info
-
-# Created by editors
+# Editors and IDEs
 *~
 \#*
 \.\#*
 *.swp
-*.ipynb_checkpoints*
+.idea/
+
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# C extensions
+*.so
+
+# Distribution / packaging
+.Python
+build/
+develop-eggs/
+dist/
+downloads/
+eggs/
+.eggs/
+lib/
+lib64/
+parts/
+sdist/
```

```suggestion
parts/
bin/
sdist/
```

I've seen this in other examples, but doesn't seem like a big deal so I will merge regardless.